cognitedata/reveal
# feat: display 2D annotations with hover events in 360 images

#### Type of change

![Feat](https://img.shields.io/badge/Type-Feat-green)  

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/REV-871
https://cognitedata.atlassian.net/browse/REV-874

## Description :pencil:


Proof of concept demonstrating automatic retrieval and display of annotations associated with 360 images.
Also implements hover events for annotations.

**Bug**: Annotations are visible while transitioning into the 360 image they belong to, which looks off. Will fix in a later PR

## How has this been tested? :mag:



- Start Reveal
- Open https://localhost:3000/?project=3d-test&env=greenfield-3d&modelId=4923307033982497&revisionId=3100284276970593
- Load 360 images named `helideck-site-2-jpeg`
- Enter the lowermost 360 image, at the edge of the helipad
- Observe that annotations associated with the images are visible
- Observe that hovering over an annotation with the mouse triggers logging of the annotation

The image annotations can be inspected and modified in Fusion: https://cog-3d.fusion.cognite.com/3d-test/vision/explore?cluster=greenfield.cognitedata.com&env=greenfield (Sort images by number of annotations to easily find the ones used in the above example).


## Test instructions :information_source:




## Checklist :ballot_box_with_check:
